apps/civil/civil-sdt/civil-sdt.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: civil-sdt
+  namespace: civil
+spec:
+  releaseName: civil-sdt
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/civil/sdt:prod-f7b9f73-20230120155915 #{"$imagepolicy": "flux-system:civil-sdt"}
```

Comment:
The image tag (prod-f7b9f73-20230120155915) is one of the civil-sdt-gateway ones.  It should be the latest civil-sdt prod image tag from the [hmctspublic.io container registry](https://portal.azure.com/#blade/Microsoft_Azure_ContainerRegistries/RepositoryBlade/id/%2Fsubscriptions%2F8999dec3-0104-4a27-94ee-6588559729d1%2FresourceGroups%2Frpe-acr-prod-rg%2Fproviders%2FMicrosoft.ContainerRegistry%2Fregistries%2Fhmctspublic/repository/civil%2Fsdt)

Reply:
Will resolve this issue once Demo branch is working fine